dynawo/sources/Modeler/Common/DYNSubModel.h

```diff
@@ -1352,6 +1432,68 @@ class SubModel {
    */
   int getOffsetY() const {return offsetY_;}
 
+  /**
+   * @brief Retrieves the local buffer for variables
+   *
+   * @returns local buffer
+   */
+  inline double* yLocal() const {
```

Comment:
We cannot return a const ponter as rvalues cannot be const and we require to retrieve the adress to perform buffer connections for external variables

Reply:
The problem will remain the same, since we'll have an API to return a pointer on the location

Reply:
still not good, I would like to avoid giving out addresses or ptr outside of the sub model, so please implement this differently. Doing the mapping in modelMulti instead of Connector for example?

Reply:
In any implementation, a sub model would have access to a location outside, because we connect buffers

Reply:
nope, not if modelMulti (which owns the full table) is in charge of doing this